R/Spectra.R

```diff
@@ -471,6 +471,23 @@ NULL
 #'   `rt[2]`. Returns the filtered `Spectra` (with spectra in their
 #'   original order).
 #'
+#' - `filterRanges`: allows filtering of the `Spectra` object based on user
```

Comment:
please update the help for `filterRanges` and `filterValues` with the updated versions suggested in `MsBackend.R` (except of the "default implementation" mention.